consul/Dockerfile

```
@@ -4,19 +4,28 @@
 FROM quay.io/cybozu/golang:1.17-focal AS build
 
 ARG CONSUL_VERSION=1.10.2
+ARG DOCKER_CONSUL_REVISION=d7b8436b3c10524e018be6676a80fe872fbfa891
 
 WORKDIR /work
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN git clone --depth=1 -b v${CONSUL_VERSION} https://github.com/hashicorp/consul /work/consul
 
+RUN curl -sSfL -o docker-entrypoint.sh https://raw.githubusercontent.com/hashicorp/docker-consul/${DOCKER_CONSUL_REVISION}/0.X/docker-entrypoint.sh
```

Don't you need to set the execute bit?

```suggestion
RUN curl -sSfL -o docker-entrypoint.sh https://raw.githubusercontent.com/hashicorp/docker-consul/${DOCKER_CONSUL_REVISION}/0.X/docker-entrypoint.sh \
    && chmod +x docker-entrypoint.sh
```